coding_schemes/Needs_Met_Yesno.json

```diff
@@ -98,6 +98,22 @@
       "DisplayText": "Need Investment",
       "VisibleInCoda": true
     },
+    {
+      "VisibleInCoda": true,
+      "NumericValue": 12,
+      "CodeID": "code-cb776c99495a",
+      "CodeType": "Normal",
+      "StringValue": "Rent",
+      "DisplayText": "Rent"
+    },
+    {
+      "VisibleInCoda": true,
+      "NumericValue": 13,
+      "CodeID": "code-0bf74d7f6cbf",
+      "CodeType": "Normal",
+      "StringValue": "Clothes",
```

Comment:
To be consistent with the others, the string values should be lower case

Reply:
Noted